File: Math/codeforces_1030A.py

# Problem link ===>> https://codeforces.com/problemset/problem/1030/A

def is_easy(responses):
  """Returns True if the problem is easy according to all responses, False otherwise."""
  return all(response == 0 for response in responses)

def main():
  n = int(input())
  responses = [int(response) for response in input().split()]

  if is_easy(responses):
    print("EASY")
  else:
    print("HARD")

if __name__ == '__main__':
  main()

File: Math/codeforces_1030A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/1030/A
// submission link ===>> https://codeforces.com/contest/1030/submission/225690503

#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;

    int easy_count = 0;
    for (int i = 0; i < n; i++)
    {
        int response;
        cin >> response;

        if (response == 0)
        {
            easy_count++;
        }
    }

    if (easy_count == n)
    {
        cout << "EASY" << endl;
    }
    else
    {
        cout << "HARD" << endl;
    }

    return 0;
}
